content/en/docs/release-oversight/disruption-testing/aggregated-disruption.md

```diff
@@ -0,0 +1,83 @@
+      4. Failure - periodic-ci-openshift-release-master-ci-4.14-upgrade-from-stable-4.13-e2e-gcp-ovn-rt-upgrade/1638769360858255360
+      5. Failure - periodic-ci-openshift-release-master-ci-4.14-upgrade-from-stable-4.13-e2e-gcp-ovn-rt-upgrade/1638769359960674304
+      6. Failure - periodic-ci-openshift-release-master-ci-4.14-upgrade-from-stable-4.13-e2e-gcp-ovn-rt-upgrade/1638769362481451008
+      7. Failure - periodic-ci-openshift-release-master-ci-4.14-upgrade-from-stable-4.13-e2e-gcp-ovn-rt-upgrade/1638769364150784000
+      8. Failure - periodic-ci-openshift-release-master-ci-4.14-upgrade-from-stable-4.13-e2e-gcp-ovn-rt-upgrade/1638769364989644800
+      9. Failure - periodic-ci-openshift-release-master-ci-4.14-upgrade-from-stable-4.13-e2e-gcp-ovn-rt-upgrade/1638769359117619200
+```
+
+you can see that the `suite=[BackendDisruption], cache-oauth-api-new-connections disruption P70 should not be worse` test failed, the P70 value is 6 seconds
+and the prow job IDs follow with the amount of disruption recorded.  This allows you to see how close the underlying jobs came to passing (in the
+above example, the values are pretty bad except for one of them which was off by 1 second).
+
+#### Disabling disruption for a specific set of tests
+
+In the past, disruption testing has uncovered some very subtle and difficult to find defects.  In those cases,
+it may take several days to find the root cause of the problem.  In a situation like this, it may make sense to
+write a bug, mark it as "blocking", and get Accepted payloads flowing again in an effort to avoid blocking development.
+In this case, you can temporarily turn off disruption testing for certain aggregated jobs.
+
+To do this, make a modification similar to this code as was done in [this PR](https://github.com/openshift/ci-tools/pull/3317/files):
+
+{{% card-code header="[ci-tools/pkg/jobrunaggregator/jobrunaggregatoranalyzer/analyzer_disruption.go](https://github.com/openshift/ci-tools/blob/01097fb7d27a7477e2f2244f8d94e3b4ba44aad7/pkg/jobrunaggregator/jobrunaggregatoranalyzer/analyzer_disruption.go#L76)" %}}
+
+```go
+			// Temporarily skip all azure disruption aggregation due to https://issues.redhat.com/browse/TRT-889
+			if strings.Contains(o.jobName, "azure") {
+				status = testCaseSkipped
+			}
+```
+
+{{% /card-code %}}
+
+The idea is to set the resulting `status` of the aggregation to "skipped".  In the example above, the code was added to
+skip disruption aggregation for tests where the job has the string "azure" in it.  A similar approach  could
+be used to skip disruption for other types of jobs.
+
+NOTE: Exercise caution when doing this because you are removing a key quality signal (in aggregated jobs) which
+means that other disruption related defects could be introduced without you knowing.  Ensure there's a plan to
+re-enable the aggregated disruption testing as soon as possible and with the understanding that once you remove it,
+you may uncover another defect.
```

Comment:
```suggestion
{{% alert title="NOTE" color="warning" %}}
Exercise caution when doing this because you are removing a key quality signal (in aggregated jobs) which
means that other disruption related defects could be introduced without you knowing.  Ensure there's a plan to
re-enable the aggregated disruption testing as soon as possible and with the understanding that once you remove it, you may uncover another defect.
{{% /alert %}}
```

Reply:
ah, thanks for that! that looks so pro.
<img width="1282" alt="Screen Shot 2023-03-24 at 4 38 13 PM" src="https://user-images.githubusercontent.com/93946516/227634147-a3f9e971-f32e-4fc0-86e0-f7b49db7e3f8.png">